src/main/java/core/basesyntax/figure/GetArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.figure;
+
+public interface GetArea {

```

Comment:
```suggestion
public interface AreaCalculator {

```

посмотри в чеклист: https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced